src/osgAnimation/MorphTransformHardware.cpp

```diff
@@ -46,23 +48,22 @@ bool MorphTransformHardware::init(MorphGeometry& morphGeometry)
     ///check for correct morph configuration
     ///(blender osgexport doesn't set sources so assume morphgeom arrays are sources)
     if(pos)
-    {
+    {   pos->setDataVariance(osg::Object::STATIC);
```

Comment:
Could you put lines of code on the line following the { to keep the code readable and consistent with the rest of the OSG.